docs/js_image_layer.md

```diff
@@ -268,10 +269,12 @@ container_image(
 | :------------- | :------------- | :------------- | :------------- | :------------- |
 | <a id="js_image_layer-name"></a>name |  A unique name for this target.   | <a href="https://bazel.build/concepts/labels#target-names">Name</a> | required |  |
 | <a id="js_image_layer-binary"></a>binary |  Label to an js_binary target   | <a href="https://bazel.build/concepts/labels">Label</a> | required |  |
-| <a id="js_image_layer-compression"></a>compression |  Compression algorithm. Can be one of `gzip`, `none`.   | String | optional |  `"gzip"`  |
-| <a id="js_image_layer-generate_empty_layers"></a>generate_empty_layers |  Generate layers even if they are empty.<br><br>Helpful when using js_image_layer with rules_docker. See https://github.com/aspect-build/rules_js/pull/1714 for more info   | Boolean | optional |  `False`  |
+| <a id="js_image_layer-compression"></a>compression |  Compression algorithm. See https://github.com/bazel-contrib/bazel-lib/blob/bdc6ade0ba1ebe88d822bcdf4d4aaa2ce7e2cd37/lib/private/tar.bzl#L29-L39   | String | optional |  `"gzip"`  |
+| <a id="js_image_layer-generate_empty_layers"></a>generate_empty_layers |  DEPRECATED. An empty layer is always generated if the layer group have no matching files.   | Boolean | optional |  `False`  |
+| <a id="js_image_layer-layer_groups"></a>layer_groups |  Layer groups to create. These are utilized to categorize files into distinct layers, determined by their respective paths. The expected format for each entry is "<key>": "<value>", where <key> MUST be a valid Bazel and JavaScript identifier (alphanumeric characters),  and <value> MAY be either an empty string (signifying a universal match) or a valid regular expression.<br><br>For better performance, it is recommended to split the large parts of a `js_binary` to have a separate layer.<br><br>The matching order for layer groups is as follows:<br><br>1. `layer_groups` are checked in order first 2. If no match is found for `layer_groups`, the `default layer groups` are checked. 3. Any remaining files are placed into the app layer.<br><br>The default layer groups are as follows and always created.<br><br><pre><code>{&#10;    "node": ".*\/js\/private\/node-patches.*\|.*\/bin\/nodejs\/.*",&#10;    "package_store_1p": "\.aspect_rules_js\/.*@0\.0\.0\/node_modules",&#10;    "package_store_3p": "\.aspect_rules_js\/.*\/node_modules",&#10;    "node_modules": "\/node_modules\/",&#10;    "app": "", # empty means just match anything.&#10;}</code></pre>   | <a href="https://bazel.build/rules/lib/dict">Dictionary: String -> String</a> | optional |  `{}`  |
```

Comment:
nit: the attribute docstring doesn't render well in the table, and even if it did, it's not a great place to give longer explanation. In these cases I've been moving this long-form doc up to the top, then just reference it in the attribute